### DISCUSSION TITLE
Request for ARM Processor Support for CrateDB RPM Installation

### DISCUSSION THREAD

**Discussion by ItsssAmar** [crate/crate#16940] [Q&A]
Dear CrateDB Team,

I hope this message finds you well. I am reaching out to inquire about the availability of the latest CrateDB RPM release for ARM processor-based operating systems.

Currently, I noticed that on the official website, only the noarch release from 2020 and the x86_64 version are available. As an ARM user, I am interested in installing the latest 2024 release of CrateDB but am unable to find a compatible RPM package.

Could you please provide guidance on whether there are plans to release an ARM-compatible version of CrateDB? If not, are there any alternative methods or workarounds to install the latest version on an ARM-based OS?

Thank you for your assistance. I look forward to your response.

**Comment by mfussenegger** [comment]
Looks like we actually build arm64 rpm packages, but they're currently not added to the repository. We'll have to extend that.
In the meantime you could do a [tarball installation](https://cratedb.com/docs/guide/install/tarball.html)

Tarballs for aarch64 linux are available under https://cdn.crate.io/downloads/releases/cratedb/aarch64_linux/

**Reply by ItsssAmar** [reply]
Thanks for the reply @mfussenegger 

When 'arm64 rpm' packages will be add into the repository?
